code/modules/atmospherics/machinery/components/unary_devices/passive_vent.dm

```diff
@@ -20,28 +20,45 @@
 /obj/machinery/atmospherics/components/unary/passive_vent/process_atmos()
 	..()
 
-	var/datum/gas_mixture/environment = loc.return_air()
-	var/environment_pressure = environment.return_pressure()
-	var/pressure_delta = abs(environment_pressure - airs[1].return_pressure())
-
-	if((environment.temperature || airs[1].temperature) && pressure_delta > 0.5)
```

Comment:
it honestly may be as simple as changing this line to `if((environment.temperature || airs[1].temperature) && (pressure_delta > 0.5 || abs(external_temperature - internal.temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER))`

Reply:
Ill test it.

Reply:
It does in fact work but very slowly.

Reply:
So if you don't mind I would prefer to have it use the method I did.

Reply:
Compare the logic you have with the logic of the heat transfering pipes.

Reply:
heat exchanging pipes are pipes and as such they interact with the pipeline datum, so it is not comparable